webpack.config.signin-connect.js

```diff
@@ -0,0 +1,39 @@
+const path = require("path");
+
+const name = "signin-connect";
+
+// define preprocessor variables
+const opts = {
+  ENV: process.env.SKYNET_MYSKY_ENV || "production",
```

Comment:
is it safe to default to `production`?

Reply:
Good point but not really relevant to this PR. I plan to refactor all the webpack scripts into a single script and we can do it then -- I'll make a f/u issue.